tests/examples/awful/wallpaper/add_screen3.lua

```diff
@@ -0,0 +1,53 @@
+--DOC_GEN_IMAGE
+--DOC_HIDE_START
+local awful =  { wallpaper = require("awful.wallpaper") }
+local wibox = require("wibox")
+local gears = {color = require("gears.color") }
+
+screen._track_workarea = true
+screen[1]._resize {x = 0, y = 0, width = 160, height = 96}
+screen._add_screen {x = 161, y = 0, width = 160, height = 96}
+screen._add_screen {x = 322, y = 0, width = 160, height = 96}
+
+require("_default_look")
+
+       local wall = awful.wallpaper {
+           screen = screen[1],
+           bg     = "#222222",
+           widget = wibox.widget {
+                {
+                    fit = function(_, width, height)
+                        return width, height
+                    end,
+                    draw = function(_, _, cr, width, height)
+                        cr:set_source(gears.color("#0000ff"))
+                        cr:line_to(width, height)
+                        cr:line_to(width, 0)
+                        cr:line_to(0, 0)
+                        cr:close_path()
+                        cr:fill()
+
+                        cr:set_source(gears.color("#ff00ff"))
+                        cr:move_to(0, 0)
+                        cr:line_to(0, height)
+                        cr:line_to(width, height)
+                        cr:close_path()
+                        cr:fill()
+                    end,
+                    widget = wibox.widget.base.make_widget()
+                },
+                {
+                    text = "Center",
+                    valign = "center",
+                    align  = "center",
+                    widget = wibox.widget.textbox,
+                },
+                widget = wibox.layout.stack
+            }
+        }
+
+--DOC_HIDE_END
+
+   wall:add_screen(screen[3])
```

Comment:
Having only this line in the usage section sounds a bit confusing. I have no idea how to make it more clear, tho... Maybe this should be entirely hidden behind a --DOC_HIDE_ALL?

Reply:
done

Reply:
I was more thinking of :

```lua
local s = screen[1] --DOC_HIDE

-- Then in the wallpaper constructor:
awful.wallpaper {
    screen = s,
    -- ...
}
```

This way the code in the example can be copied/pasted directly, as we consider (from the default awesomerc.lua) that the "request::wallpaper" callback have the screen as first parameter named s.

Reply:
Well, this is the `add_screen` example. So I think it makes sense to use `add_screen`?

Reply:
Oh wait, I replied on the wrong thread... Sorry